homeassistant/components/salus/config_flow.py

```diff
@@ -0,0 +1,123 @@
+"""Config flow for user setup."""
+
+import logging
+from typing import Any, Dict
+
+from requests import ConnectTimeout, HTTPError
+from salus.api import Api
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_DEVICE, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers.typing import HomeAssistantType
+
+# pylint: disable=unused-import
```

Comment:
No longer needed. You might need to rebase onto the latest dev in order to make that work.

```suggestion
```